questions/javascript-questions.md

```diff
@@ -1,66 +1,54 @@
 # JS Questions:
 
-* Explain event delegation
-* Explain how `this` works in JavaScript
-* Explain how prototypal inheritance works
-* What do you think of AMD vs CommonJS?
-* Explain why the following doesn't work as an IIFE: `function foo(){ }();`.
-  * What needs to be changed to properly make it an IIFE?
+* Explain event delegation.
+* Explain how `this` works in JavaScript.
+  * Can you give an example of how `this` has changed in ES6?
```

Comment:
I don't fancy the wording, I'd suggest something like 

> Can you give an example of how things involving `this` have changed with ES6?

Reply:
`this` is common term - why not?

Reply:
I mean that `this` didn't actually change in ES6 but things around `this` did. 
Just a simple semantic remark.

Reply:
@GMaiolo I think it's a valid comment. I'll adjust the wording.